packages/uploadthing/src/mime-types/index.ts

```diff
@@ -71,8 +71,8 @@ function populateMaps(
   // source preference (least -> most)
   const preference = ["nginx", "apache", undefined, "iana"];
 
-  Object.keys(db).forEach(function forEachMimeType(type) {
-    const mime = db[type];
+  Object.keys(mimeDB).forEach(function forEachMimeType(type) {
+    const mime = mimeDB[type as MimeType] as unknown as MimeObject;
```

Comment:
these type casts shouldn't be necessary?

Reply:
It got very angry when I didn't have them. Will play with it some more though